subprojects/zap-clientapi/src/main/java/org/zaproxy/clientapi/core/ClientApi.java

```diff
@@ -361,13 +361,16 @@ private static URL buildZapRequestUrl(
         sb.append('/');
         if (params != null) {
             sb.append('?');
+	    String amp = "";

```

Comment:
Indentation wrong here and below? We do have a nasty mix of spaces and tabs at the moment I'm afraid :/